data_validation/config_manager.py

```diff
@@ -944,6 +946,16 @@ def require_pre_agg_calc_field(
                         f"Skipping {agg_type} on {column} due to data type: {column_type}"
                     )
                 continue
+            # Oracle LOB columns are invalid for use with SQL COUNT function.
+            elif agg_type == "count" and self._is_oracle_lob(column):
+                # TODO Eventually we need to allow COUNT(Oracle LOB) by using a CASE expression:
+                #      COUNT(CASE WHEN clob_col IS NULL THEN NULL ELSE 1 END)
```

Comment:
I don't think I can skip it in the query builder because I don't have data type information for both the source and target. I believe ConfigManager is where I have all of the information required to exclude a column from both source and target queries.